INSTALL.rst

```diff
@@ -1,156 +1,230 @@
+
+How you install ``scikit-image`` depends on your needs and skill:
+
+- Simplest solution:
+  `a scientific Python distribution <#scientific-python-distributions>`_.\
+
+- If you can install Python packages and work in virtual environments:
+
+  - `pip <#install-via-pip>`_
+
+  - `conda <#install-via-conda>`_
+
+- Easy but has pitfalls: `system package manager (yum, apt-get,...) <#system-package-manager>`_
+
+- `You're looking to contribute to scikit-image <#building-from-source>`_
+
+Supported platforms
+------------------------------------------------------------------------------
+
+- Windows 64-bit on x86 processors
+- Mac OS X on x86 processors
+- Linux 64-bit on x86 processors
+
+For information on other platforms, see `Other platforms <#other-platforms>`_.
+
+Version check
+------------------------------------------------------------------------------
+
+To see whether ``scikit-image`` is already installed or to check if an install has
+worked, run the following in a Python shell or Jupyter notebook:
+
+.. code-block:: python
+
+  import skimage
+  print(skimage.__version__)
+
+or, from the command line
+
+.. code-block:: sh
+
+   python -c "import skimage; print(skimage.__version__)"
+
+(Try ``python3`` if ``python`` is unsuccessful.)
+
+You'll see the version number if ``scikit-image`` is installed and
+an error message otherwise.
+
+Scientific Python distributions
+------------------------------------------------------------------------------
+
+In a single install these give you Python,
+``scikit-image`` and libraries it depends on, and other useful scientific
+packages. They install into an isolated environment, so they won't conflict
+with any existing installed programs.
+
+Drawbacks are that the install can be large and you may not get
+the most recent ``scikit-image``.
+
+We recommend one of these distributions:
+
+- `Anaconda <https://www.anaconda.com/distribution/>`_
+- `Python(x,y) <https://python-xy.github.io/>`_
+- `WinPython <https://winpython.github.io/>`_
+
+When using the ``scikit-image``
+documentation, make sure it's for the version you've installed (see
+`Version check <#version-check>`_ above).
+
+
+Installation via pip and conda
+------------------------------------------------------------------------------
+
+These install only ``scikit-image`` and its dependencies; pip has an option to
+include related packages.
+
+.. _install-via-pip:
+
+pip
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Prerequisites to a pip install: You're able to use your system's command line to
+install packages and are using a
+`virtual environment \
+<https://towardsdatascience.com/virtual-environments-104c62d48c54?gi=2532aa12906#ee81>`_
+(any of
+`several \
+<https://stackoverflow.com/questions/41573587/what-is-the-difference-between-venv-pyvenv-pyenv-virtualenv-virtualenvwrappe>`_\
+).
+
+Nothing will stop you from using pip without a virtual environment,
```

Comment:
Maybe we can remove the reference here and say something like: "While it is possible to use pip without a virtual environment, it is not advised: they create a clean Python environment that does not interfere with any existing system installation, can be easily removed, and contains only the packages that you need."